infer_structcol/model.py

```diff
@@ -12,6 +12,10 @@
 # define limits of validity for the MC scattering model
 min_phi = 0.35
 max_phi = 0.73
+min_radius = 70. # in nm
```

Comment:
Changed the values to be: 
min_phi = 0.1
max_phi = 0.74
min_radius = 10 nm
max_radius = 1000 nm